src/vm/eetoprofinterfaceimpl.cpp

```diff
@@ -2053,9 +2053,9 @@ HRESULT EEToProfInterfaceImpl::EnsureProfilerDetachable()
 }
 
 // Declarations for asm wrappers of profiler callbacks
-EXTERN_C void __stdcall ProfileEnterNaked(FunctionIDOrClientID functionIDOrClientID);
-EXTERN_C void __stdcall ProfileLeaveNaked(FunctionIDOrClientID functionIDOrClientID);
-EXTERN_C void __stdcall ProfileTailcallNaked(FunctionIDOrClientID functionIDOrClientID);
+EXTERN_C void ProfileEnterNaked(FunctionIDOrClientID functionIDOrClientID);
+EXTERN_C void ProfileLeaveNaked(FunctionIDOrClientID functionIDOrClientID);
+EXTERN_C void ProfileTailcallNaked(FunctionIDOrClientID functionIDOrClientID);
```

Comment:
Did these just become __cdecl on windows x86 now? Sorry I don't recall what the windows compiler emits when you aren't specific.

Reply:
This change has no impact on Windows. We compile with stdcall as the default convention on Windows. Many places depend on this default.